build.sh

```diff
@@ -13,9 +13,9 @@ if [ -z "${OSU_USE_SYSTEM_COMPOSER:-}" ]; then
   COMPOSER="php composer.phar"
 
   if [ -f composer.phar ]; then
-    php composer.phar self-update --1
+    php composer.phar self-update --2
   else
-    curl -sL https://getcomposer.org/composer-1.phar > composer.phar
+    curl -sL https://getcomposer.org/download/latest-stable/composer.phar > composer.phar
```

Comment:
maybe worth pinning to 2.x?

`https://getcomposer.org/download/latest-2.x/composer.phar`